spring-kafka/src/test/java/org/springframework/kafka/listener/KafkaMessageListenerContainerTests.java

```diff
@@ -2486,6 +2486,34 @@ public void onPartitionsAssigned(Collection<TopicPartition> partitions) {
 		logger.info("Stop rebalance after failed record");
 	}
 
+	@Test
+	void enforceRabalanceOnTheConsumer() throws Exception {
+		ConsumerFactory<Integer, String> cf = mock();
+		ContainerProperties containerProps = new ContainerProperties("foo");
```

Comment:
I said before that I don't like non-official language in open source.
Please, consider to use some other words.
Even if we have it somewhere in the code already, that doesn't mean we should continue to use what might be harmful one day.

Thanks